src/components/sidebar/MenuMap.js

```diff
@@ -63,9 +73,17 @@ class MenuMap extends Component {
       </div>
     )
   }
+  menuToggle = (val, props) => {
+      const o = val.filter((v) => {
+         return v.key === props.name
+      });
+      return o.length ? o[0].val : 'open';
+  }
   render() {
+    const {menuOpen} = this.props;
+    const v = this.menuToggle(menuOpen, this.props) == 'open'? true : false;
```

Comment:
if we make [this](https://github.com/no-stack-dub-sack/cs-playground-react/pull/66/files#r171135826) change, mentioned above, we can just do `const v = this.menuToggle(menuOpen, this.props)`. And also, let's avoid abbreviated variable names in general. It's nice in terms of brevity, but it makes the code harder to read and understand for someone else taking a look. Let's call it `value` or something like `menuState` instead.

Reply:
Done